set.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+)
+
+type JsonSet struct {
+	// The Consul Key to set.
+	Key string
+	// The value that was passed by the command line.
+	value string
+	// The actual Json Value that is going to be saved to Consul.
+	JsonValue []byte
+}
+
+func (js *JsonSet) ParseFlags(args []string) {
+	if len(args) < 2 {
+		fmt.Println("jsonconsul set full/key/path \"true\"")
+		os.Exit(-1)
+	}
+
+	js.Key = args[0]
+	js.value = args[1]
+}
+
+func (js *JsonSet) lintedJson() []byte {
```

Comment:
We should probably ensure the value is either a bool or float at this point as well.